packages/language-model/src/costs/model-prices.ts

```diff
@@ -79,7 +79,7 @@ export const openAiTokenPricing: ModelPriceTable = {
 			},
 		],
 	},
-	o3: {
+	"o3": {
```

Comment:
Let me apply this boy-scout change to unify format to other models (although the original code works)